ui/Screen/RegisterUser.svelte

```diff
@@ -0,0 +1,103 @@
+<script>
+  import { ModalLayout, StepCounter } from "../DesignSystem/Component";
+
+  import PickHandleStep from "./RegisterUser/PickHandleStep.svelte";
+  import SubmitRegistrationStep from "./RegisterUser/SubmitRegistrationStep.svelte";
+
+  import { Text, Title } from "../DesignSystem/Primitive";
+  import { gql } from "apollo-boost";
+  import { getClient, mutate } from "svelte-apollo";
+  import { pop } from "svelte-spa-router";
```

Comment:
Would group third-party imports at the top and group owned ones closer to the implementation.